admin-dev/themes/new-theme/js/app/utils/init-components.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+import TranslatableField from '@js/components/translatable-field.js';
+import TranslatableInput from '@js/components/translatable-input.js';
+import TinyMCEEditor from '@js/components/tinymce-editor.js';
+import TaggableField from '@js/components/taggable-field.js';
+
+const initComponents = () => {
+  if (window.prestashop === undefined) {
+    window.prestashop = {};
+  }
+
+  window.prestashop.component = {
+    initComponents(components) {
+      if (window.prestashop.instance === undefined) {
+        window.prestashop.instance = {};
+      }
+      components.forEach((component) => {
+        if (window.prestashop.instance[component] === undefined
+          && window.prestashop.component[component] !== undefined) {
+          window.prestashop.instance[component] = new window.prestashop.component[component]();
```

Comment:
Maybe we should add some `console.error` or `console.warning` when the component is not managed?

Reply:
Great suggestion! I think error makes more sense, at least for me I constantly get a couple of warnings so I think a lot of people wouldn't notice it, while the error is clearly noticeable.